### PR TITLE
+ more tv ch

### PR DESCRIPTION
**Pull request by BBBURENEEBB**
more tv chs added

### CODE REVIEW

**Comment by BBBURENEEBB** [comment]
nnnn

File: streams/mn.m3u [hunk 1]
```diff
@@ -3,6 +3,10 @@
 https://live.mnb.mn/live/mnb_sport.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="MNBWorld.mn" status="online",MNB world
 https://live.mnb.mn/live/mnb_world.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="NTV.mn" status="online",NTV
+https://cdn2.skygo.mn/live/disk1/NTVHD/DASH-FTA/NTVHD-avc1_1000000=3-2367042480000000.mp4
```
Comment (LaneSh4d0w) [inline streams/mn.m3u:7]:
```suggestion
https://cdn2.skygo.mn/live/disk1/NTVHD/DASH-FTA/NTVHD.mpd
```

Comment (LaneSh4d0w) [reply streams/mn.m3u:7]: HLS would be https://cdn2.skygo.mn/live/disk1/NTVHD/HLS-FTA/NTVHD.m3u8

File: streams/mn.m3u [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+#EXTINF:-1 tvg-id="StarTV.mn" status="online",Star TV
+https://cdn2.skygo.mn/live/disk1/Star/DASH-FTA/Star-avc1_1000000=4-2367044310000000.mp4
```
Comment (LaneSh4d0w) [inline streams/mn.m3u:9]:
```suggestion
https://cdn2.skygo.mn/live/disk1/Star/DASH-FTA/Star.mpd
```

Comment (LaneSh4d0w) [reply streams/mn.m3u:9]: HLS would be https://cdn2.skygo.mn/live/disk1/Star/HLS-FTA/Star.m3u8

File: streams/mn.m3u [hunk 3]
```diff
@@ -11,3 +15,9 @@ https://live.mnb.mn/live/mnb.stream/playlist.m3u8
 http://d1qwew1jz60jqg.cloudfront.net/MYASIATV/TV5/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",MNB Гэр бүл
 https://live.mnb.mn/live/mnb_family.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="UBS.mn" status="online",UBS
+https://cdn1.skygo.mn/live/disk1/UBSHD/DASH-FTA/UBSHD-avc1_600000=1-2367045670000000.mp4
```
Comment (LaneSh4d0w) [inline streams/mn.m3u:19]:
```suggestion
https://cdn1.skygo.mn/live/disk1/UBSHD/DASH-FTA/UBSHD.mpd
```

Comment (LaneSh4d0w) [reply streams/mn.m3u:19]: https://cdn1.skygo.mn/live/disk1/UBSHD/HLS-FTA/UBSHD.m3u8 for HLS

File: streams/mn.m3u [hunk 4]
```diff
@@ -11,3 +15,9 @@ https://live.mnb.mn/live/mnb.stream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="TV25.mn" status="online",MN25
+https://cdn2.skygo.mn/live/disk1/MN25/DASH-FTA/MN25-avc1_1000000=2-2367046720000000.mp4
```
Comment (LaneSh4d0w) [inline streams/mn.m3u:21]:
```suggestion
https://cdn2.skygo.mn/live/disk1/MN25/DASH-FTA/MN25.mpd
```

Comment (LaneSh4d0w) [reply streams/mn.m3u:21]: https://cdn2.skygo.mn/live/disk1/MN25/HLS-FTA/MN25.m3u8 for HLS

File: streams/mn.m3u [hunk 5]
```diff
@@ -11,3 +15,9 @@ https://live.mnb.mn/live/mnb.stream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="BolovsrolTV.mn" status="online",Education
+https://cdn3.skygo.mn/live/disk1/Education/DASH-FTA/Education-avc1_1000000=3-2367048879877000.mp4
```
Comment (LaneSh4d0w) [inline streams/mn.m3u:23]:
```suggestion
https://cdn3.skygo.mn/live/disk1/Education/DASH-FTA/Education.mpd
```

Comment (LaneSh4d0w) [reply streams/mn.m3u:23]: https://cdn3.skygo.mn/live/disk1/Education/HLS-FTA/Education.m3u8 for HLS

**Comment by BBBURENEEBB** [comment]
approv pls billions

**Review by BBBURENEEBB — COMMENTED** [review]
rev

**Comment by BBBURENEEBB** [comment]
rev

**Review by BellezaEmporium — DISMISSED** [review]

**Review by its0x08 — DISMISSED** [review]

**Review by what-a — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**